pkgs/games/marble-marcher-ce/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, stdenv
+, makeWrapper
+, makeDesktopItem
+, copyDesktopItems
+, fetchFromGitHub
+, sfml
+, anttweakbar
+, glm
+, eigen
+, glew
+, cmake
+}:
+stdenv.mkDerivation rec {
+  pname = "marble-marcher-ce";
+  version = "1.4.5";
+
+  src = fetchFromGitHub {
+    owner = "WAUthethird";
+    repo = "Marble-Marcher-Community-Edition";
+    rev = version;
+    sha256 = "sha256-m5i/Q4k5S4wcojHqMYS7e1W/Ph7q/95j3oOK2xbrHSk=";
+  };
+
+  buildInputs = [ sfml anttweakbar glm eigen glew ];
+  nativeBuildInputs = [ cmake makeWrapper copyDesktopItems ];
+  installFlags = [ "DESTDIR=$(out)" ];
+
+  postInstall = ''
+    mkdir $out/bin
+    mkdir -p $out/share/icons/
+    makeWrapper $out/home/MMCE/MarbleMarcher $out/bin/${pname} --chdir $out/home/MMCE
```

Comment:
What's this all about? Please move the game file into a sensible place. Why do we need to chdir?

Reply:
Unfortunately the executable has to be ran from the same folder the assets are.

Reply:
Alright, please add that as a comment.

I'm not 100% sure where the assets should live but `home/` isn't it. I'd be happy with `opt/` but that's a bit too FHS-y for my liking.  
You're far more involved with native game packaging than I am, where do games and assets usually live?

Reply:
See also https://github.com/WAUthethird/Marble-Marcher-Community-Edition/issues/23 for the weird path.

Reply:
> You're far more involved with native game packaging than I am, where do games and assets usually live?
Usually just `share`

Reply:
I see two options. Move the assets to `share` in `postInstall` or patch the CMakefile to change `home` to `share`
Which one would you prefer @Atemu ?

Reply:
I went ahead and patched the `CMakeLists.txt`
As @lboklin points out in [this](https://github.com/WAUthethird/Marble-Marcher-Community-Edition/issues/23#issuecomment-975344162) comment this means that the screenshots can't be saved and probably progress.

Reply:
I think not being able to save screenshots isn't that big of an issue but I looked into it and it can be patched easily to save the screenshots in the config directory [here](https://github.com/WAUthethird/Marble-Marcher-Community-Edition/blob/9de1763f65ebad109245843c516cd2c5ae2edc9c/src/Gamemodes.cpp#L966):